src/features/content-explorer/content-explorer/__tests__/ContentExplorerHeaderActions.test.js

```diff
@@ -92,6 +100,40 @@ describe('features/content-explorer/content-explorer/ContentExplorerHeaderAction
             expect(wrapper.instance().props.customInput).toEqual(customInput);
             expect(wrapper).toMatchSnapshot();
         });
+
+        test('should pass foldersPath to ContentExplorerFolderTreeBreadcrumbs', () => {
```

Comment:
I think it's okay to omit these tests that are just checking for passthrough props. this would be hard to maintain if we tried to do this for all components and props